### ISSUE TITLE
Removed USE_EXACT_ALARAM still getting error "You must let us know whether your app uses any exact alarm permissions"

### ISSUE THREAD

**Issue by iamongit** [MaikuB/flutter_local_notifications#2348]
[Android] I had both SCHEDULE_EXACT_ALARM and USE_EXACT_ALARM in my manifest. 
Updated the plugin to **17.1.2** and removed USE_EXACT_ALARM from manifest and still getting the error **"You must let us know whether your app uses any exact alarm permissions"** on trying to upload the aab to play store using the upload artifacts action.

On uploading the aab manually and trying to create a release I get a warning that I'm using USE_EXACT_ALARM which I am not. Only place I see it is in the example app of this plugin. Don't see a way past this unless I  select the app is Calendar or Alarm app which it is not. I still have SCHEDULE_EXACT_ALARM in my manifest and requesting the user for the permission as the app needs to notify the user precisely on scheduled times.

Am I missing something?

<img width="1909" alt="Screenshot 2024-06-20 at 10 39 32 PM" src="https://github.com/MaikuB/flutter_local_notifications/assets/18608258/6199f853-6eb0-462c-ab27-6da2a036648d">

## On trying to complete declaration:
<img width="1068" alt="Screenshot 2024-06-20 at 10 39 41 PM" src="https://github.com/MaikuB/flutter_local_notifications/assets/18608258/301a3e10-5574-4c9a-9fae-5d0fe23f59fb">

## UPDATE
I added the following line to my manifest but I am still getting the same error:
```<uses-permission android:name="android.permission.USE_EXACT_ALARM" tools:node="remove" />```

**Comment by crushman1** [comment]
what is the solution to this ?

**Comment by iamongit** [comment]
> what is the solution to this ?

Depends, if your app qualifies to use USE_EXACT_ALARM then upload the aab manually and complete the declaration you'll find on trying to send the release changes for review, select if the core functionality revolves around it being a calendar or alarm app which hopefully doesn't change the app category and will be reviewed by google thoroughly. I completed the declaration 2 days ago and it's still under review so not sure what will come out of it, our app needs to schedule exact alarms but it's a medical app, not an Alarm or Calendar app.

If that doesn't work or apply then you'll have to wait for the plugin to update or fork it and remove the USE_EXACT_ALARM permission and reference the dependency using git instead of pub package.

SCHEDULE_EXACT_ALARM will let you use exact alarm times but you'll have to request the user for that permission.

**Comment by istornz** [comment]
Same issue on my side...

**Comment by MaikuB** [comment]
Fyi the plugin has been updated since 16.0. It is open source so you can verify this as well https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/android/src/main/AndroidManifest.xml. Have an app at work that uses the plugin as a transitive dependency and doesn't have the issue either and seen others here report that updating to newer versions of the plugin resolved their issue

**Comment by istornz** [comment]
Ok so I found the problem and this is not the cause of the plugin 👍 

In my case, I had some old release in closed tested which have USE_EXACT_ALARM permission. I uploaded a new binary without the USE_EXACT_ALARM permission and replaced it.
To check if this is the case for you @iamongit @crushman1 , just go to "App content" in the PlayStore console and you will see red flag with the detailed appbundle with the permission.

**Comment by iamongit** [comment]
@istornz Thanks Dimitri. Yeah this was not made clear on Play Store, there was nothing on the dashboard, no email, and the only place I saw this was on trying to upload the aab via git workflow or at the time of sending the version to review for publishing. 

I already completed the declaration and it's under review for a few days now. For now while the declaration is under review I don't see this error anymore. But yeah we did have an older version in closed testing with USE_EXACT_ALARM permission which could have been causing the issue as you described, makes sense.
